spec/lib/api/login/login-spec.js

```diff
@@ -81,7 +81,7 @@ describe('Http.Api.Login', function () {
     before('setup default admin user', function() {
         waterline = helper.injector.get('Services.Waterline');
         waterline.localusers = {
-            findOne: sinon.stub()
+            findOne: sandbox.stub()
```

Comment:
For this specific file, it defines its own sandbox, so we keep it here. I agree with you that it's better to align with other files, but it needs more work.